browser.js

```diff
@@ -1,78 +1,86 @@
-import { CallbackList } from 'meteor/flowkey:callback-list';
-import { Blaze } from 'meteor/blaze';
+import { CallbackList } from "meteor/flowkey:callback-list";
+import { Blaze } from "meteor/blaze";
+import { enhanceWithPromiseSupport } from "./enhanceWithPromiseSupport";
 
-import './flow-notification.html';
+import "./flow-notification.html";
 
-class Notification{
-	constructor(){
-		this.running = false;
-		this.callbacks = new CallbackList();
-	}
+class Notification {
+    constructor() {
+        this.running = false;
+        this.callbacks = new CallbackList();
+    }
 
-	alert({message, callback, title = 'Alert', buttonLabel = 'Ok'}){
-		if (!this.running){
-			this.callback = callback;
-			this.view = Blaze.renderWithData(Template.__FlowAlert, {title, message, buttonLabel}, document.body);
-			this.running = true;
-		}else{
-			// delay other alerts till running one is closed
-			this.callbacks.add(()=>{
-				this.alert.apply(this, arguments);
-			})
-		}
-	}
+    alert({ message, callback, title = "Alert", buttonLabel = "Ok" }) {
+        if (!this.running) {
+            this.callback = callback;
+            this.view = Blaze.renderWithData(
+                Template.__FlowAlert,
+                { title, message, buttonLabel },
+                document.body
+            );
+            this.running = true;
+        } else {
+            // delay other alerts till running one is closed
+            this.callbacks.add(() => {
+                this.alert.apply(this, arguments);
+            });
+        }
+    }
 
+    confirm({ message, callback, title = "Confirm", buttonLabels = ["Ok", "Cancel"] }) {
+        if (!_(buttonLabels).isArray()) throw new Meteor.Error("Button Label has to be an array");
+        if (!this.running) {
+            this.callback = callback;
+            this.view = Blaze.renderWithData(
+                Template.__FlowAlert,
```

Comment:
It'd be Ideal if we could import this rather than relying on a global, but I'm not sure if Blaze can deal with that?

Reply:
I didn't change any of this code here. A quick look in the blaze documentation suggested that this isn't possible